README.md

```diff
@@ -1008,6 +1019,7 @@ Using **CONAN_CLANG_VERSIONS** env variable in Travis ci or Appveyor:
 - **mingw_configurations**: Configurations for MinGW
 - **archs**: List containing specific architectures to build for. Default ["x86", "x86_64"]
 - **use_docker**: Use docker for package creation in Linux systems.
+- **docker_build_options**: Pass additional parameters for docker when running the create step.
```

Comment:
Is it relevant for CI? If yes, I would say to include environment variable support.

Reply:
The support was already there, but I forgot to mention it readme. Fixed.